### PR TITLE
AC Automated Test - LRAC-8590 | master

### PR DESCRIPTION
**Pull request by alanafergs**
https://issues.liferay.com/browse/LRAC-8590

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by timpak** [comment]
reviewing PR

File: portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACUtils.macro [hunk 1]
```diff
@@ -231,6 +231,13 @@ definition {
 		Click(locator1 = "ACUtils#CLEAR_SEARCH_BAR_BUTTON");
 	}
 
+	macro clickAnyButton {
+		var buttonName = "${button}";
```
Comment (timpak) [inline portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACUtils.macro:235]: hey @alanafergs a new line is needed after the var to make things more legible

File: portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACUtils.path [hunk 2]
```diff
@@ -89,6 +89,10 @@
 <tr>
 	<td>DROPDOWN_ITEM</td>
 	<td>//div[contains(@class, 'dropdown-menu show')]//*[text()='${key_dropdownItem}']</td>
+</tr>
+<tr>
+	<td>GENERIC_BUTTON</td>
+	<td>//button[contains(@class,'button-root')and text()='${key_buttonName}']</td>
```
Comment (timpak) [inline portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACUtils.path:95]: The previous path, DROPDOWN_ITEM, is missing <td></td> after line 91. This should fix the current poshi validation error.

File: portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACUtils.path [hunk 3]
```diff
@@ -89,6 +89,10 @@
 <tr>
 	<td>DROPDOWN_ITEM</td>
 	<td>//div[contains(@class, 'dropdown-menu show')]//*[text()='${key_dropdownItem}']</td>
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACUtils.path:91]: Hi @alanafergs, put `<td></td>` after this line

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EditSegments.testcase [hunk 4]
```diff
@@ -477,9 +477,85 @@ definition {
 	@ignore = "true"
 	@priority = "4"
 	test UndoAddingSegmentMemberInTheEditModal {
+		task ("Get site assigned property name") {
+			var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+		}
+
+		task ("Add a new user in DXP") {
+			for (var name : list "ac,liferay") {
```
Comment (timpak) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EditSegments.testcase:485]: I would extract the list `ac,liferay` to a `var` and call that `var` when needed.

**Comment by alexcandidos** [comment]
hi @alanafergs  , I'm closing this PR for now, please open a new one when it is ready. Thanks!

**Pull request closed without merging**